backend/oasst_backend/journal_writer.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, db: Session, api_client: ApiClient, user: User):
         self.user = user
         self.user_id = self.user.id if self.user else None
 
-    def log_text_reply(self, task: Task, message_id: UUID, role: str, length: int) -> Journal:
+    def log_text_reply(self, task: Task, message_id: Optional[UUID], role: str, length: int) -> Journal:
```

Comment:
not clear to me why the message_id became Optional now, it is essential for the journal entries. We need to clarify this via discord.